adapters/repos/db/lsmkv/segment_bloom_filters_test.go

```diff
@@ -48,6 +47,8 @@ func TestCreateBloomOnFlush(t *testing.T) {
 
 	_, ok = findFileWithExt(files, "secondary.0.bloom")
 	assert.True(t, ok)
+	// on Windows we have to shutdown the bucket before opening it again
```

Comment:
Why is this?

Reply:
@ahmedalhulaibi can you answer this?

Reply:
On windows the file is locked when open IIRC

Reply:
Thanks for the explanation @ahmedalhulaibi